sdk/appconfiguration/app-configuration/package.json

```diff
@@ -97,37 +85,24 @@
     "tslib": "^2.2.0"
   },
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.1.0",
-    "@azure-tools/test-utils": "^1.0.1",
```

Comment:
If you need it, you can use `test-utils-vitest` for just about everything test-utils offered

Reply:
I realized we don't have any import from "test-utils" so the dependency is removed